packages/models-library/src/models_library/basic_types.py

```diff
@@ -34,6 +34,11 @@ class MD5Str(ConstrainedStr):
     regex = re.compile(r"^[a-fA-F0-9]{32}$")
 
 
+# sha256sum path/to/file
```

Comment:
MINOR: put both SHA*Str one next to the other

Reply:
Done